docs/source/conf.py

```diff
@@ -333,6 +333,7 @@ def get_incollection_template(self, e):
     "examples/mitiq-paper/*": "_static/mitiq-codeblocks.png",
     "examples/zne-braket-ionq": "_static/zne-braket-ionq.png",
     "examples/bqskit": "_static/bqskit.png",
+    "examples/learning-depolarizing-noise": "_static/learn-depolarizing.png",
```

Comment:
It looks like the thumbnail image needs to be moved into `docs/source/_thumbnails` in order to show on the gallery page.

Reply:
I thought it was just a rendering issue on my browser, but moving into `docs/source/_thumbnails` was the right call.